crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -49,7 +49,11 @@ impl Epoch {
             .fetch_sui_system_state(Some(self.stored.epoch as u64))
             .await?;
 
-        let active_validators = convert_to_validators(system_state.active_validators, None);
+        let active_validators = convert_to_validators(
```

Comment:
I must admit that I do not fully follow this change with needing checkpoints. The active validators are the same throughout an epoch, so to me the last_checkpoint_id is not needed. It's probably somehow needed later in the chain, but fail to see it yet.

Reply:
Thanks for reviewing! Yes it's hard to contextualize the changes here, and perhaps there's another way to achieve the end goal. The reason I add the chkpt as a field on Validator is so that it can pass the chkpt to Address when someone tries to resolve for ReportRecord or another Address field on validator. 
If I fetch system state for a historical epoch and view the active validators at that epoch, the address fields would need that chkpt context, otherwise we will always show the latest - which would be incorrect